cloud/cloud.py

```diff
@@ -82,8 +82,11 @@ def init_resources(self, cloud_config):
             self)
         self.resources['identity'] = identity
 
+        disabled_resources = ['identity']
```

Comment:
Actually, 'identity' is not disabled, It goes separate because another resource depend on the already initialized identity resource.